controllers/mealFoodsController.js

```diff
@@ -2,86 +2,82 @@ const Meal = require('../models').Meal
 const User = require('../models').User
 const Food = require('../models').Food
 const MealFood = require('../models').MealFood
-const pry = require('pryjs')
+const pry = require('pryjs');
 
-const create = (req, res) => {
-  User.findOne({ where: { api_key: req.body.api_key } })
-    .then(user => {
-      const meal = sanitizeEntry(req.body.meal_name)
-      return findMeal(user, meal, req, res)
-    })
-    .catch(error => {
-      res.setHeader("Content-Type", "application/json");
-      res.status(401).send(JSON.stringify({error: "Invalid credentials."}))
-    })
+const create = async (req, res) => {
+  try {
+    const user = await User.findOne({ where: { api_key: req.body.api_key } });
+    const meal = sanitizeEntry(req.body.meal_name);
```

Comment:
Peregrine, I formally request this to become a query param instead of a body attribute. I request it of all the body attributes, in fact.